### DISCUSSION TITLE
Increasing Memory of CF FLYWAY Resource

### DISCUSSION THREAD

**Discussion by CloudandReact** [concourse/concourse#6494] [Help & Support] [answered]
Hi,
Hope you are doing well.
I am trying to increase the memory from 256MB =>1GB for CF FLYWAY https://hub.docker.com/r/emeraldsquad/cf-flyway-resource
The problem is that in my pipeline there does not seem to be an easy way to overwrite the memory.
There isn’t a default configuration in the cf flyway resource for modifying the memory.
I could manually change it in Cloud Foundry once deployed but would rather if I could somehow configure it in a pipeline through a script or something.

It seems that I cannot overwrite the Cf Flyway manifest.

Any suggestions?

**Comment by taylorsilva** [comment, marked as answer]
Hey, seems like you're struggling to find an answer to this. Looks like the repo for this resource used to be at https://github.com/emerald-squad/cf-flyway-resource but now it 404's. Maybe you can ask for help in one of their related repo's in their org https://github.com/emerald-squad/

I looked at some of the users that commit to that repo and it looks like "emerald-squad" is a team at Desjardins, a financial company in Canada.

Looking at their docs, it appears that they do not provide a way for you to modify the memory for the deployed flyway app. You could add an step right after you use this resource to adjust the memory of the deployed flyway app. Good luck!

**Reply by CloudandReact** [reply]
Thank you very much for your help will try to see if I can modify the memory like you suggested.
